data/dataset/redis_dataset.yml

```diff
@@ -0,0 +1,119 @@
+                fidesops_meta:
+                  data_type: string
+          - name: EN_MANUAL_INPUT__<privacy_request_id>__<dataset_name>:<collection_name>  # Usage: To store data uploaded by a human for a given manual collection which can be used to query downstream dependent results and later filtered and returned to the user. This is for the Manual Connector, which is integrated with the graph.
+            fidesops_meta:
+              data_type: object[]  # A list of rows manually uploaded for a given collection.  The fields will be entirely dependent on the collection in question.
+          - name: EN_MANUAL_MASK__<privacy_request_id>__<dataset_name>:<collection_name>  # Usage: To have an erasure for a Manual Connector (which is integrated with the graph) return the same information that an automated connector returns. This allows a user to confirm they've manually masked data on their end for a given collection.
+            data_categories: [system.operations]
+            fidesops_meta:
+              data_type: integer  # A count of records manually erased for a given collection
+          - name: EN_PAUSED_LOCATION__<privacy_request_id>  # Usage: Caches the location where a privacy request is paused and awaiting input by a manual connector. Manual data should be uploaded corresponding to this cached location.  Manual connectors are integrated with the graph.
+            fidesops_meta:
+              data_type: object
+              fields:
+                - name: step  # Either access or erasure
+                  data_categories: [system.operations]
+                  fidesops_meta:
+                    data_type: string
+                - name: collection # dataset_name:collection_name.  The given collection where the manual connector is awaiting input.
+                  data_categories: [system.operations]
+                  fidesops_meta:
+                    data_type: string
+                - name: action_needed
+                  fidesops_meta:
+                    data_type: object[]
+                  fields:
+                    - name: locators
+                      data_type: object  # Field names mapped to values that should be used to manually locate the record(s) associated with a given subject
+                    - name: get  # Applicable if we're paused on the "access" step. A list of field names on the current collection that should be retrieved.
+                      data_categories: [user]
+                      data_type: string[]
+                    - name: update  # Applicable if we're paused on the "erasure" step
+                      data_type: object # Field names mapped to the values that should be used to mask
+          - name: EN_PLACEHOLDER_RESULTS__<privacy_request_id>__access_request__<dataset_name>:<collection_name>  # Usage: Saves the access data that we should use to perform the erasure request.  Access request data is a prerequisite to running an erasure.
```

Comment:
The purpose of this key is a little tricky to explain. If an array field is returned as part of an access request, it is possible that only some of that array is relevant to the given data subject.  We can filter out the array elements that don't belong to the data subject, therefore, decreasing the size of the array.  

Access results used to be directly used to fulfill and erasure request, but this new representation was added when we started supporting array retrieval/masking.  This separately stores a version of the access results which stores an array of the original size, just with non-subject data replaced with a fidesops-specific key.  This helps us figure out which indices should be masked.